### ISSUE TITLE
[Crash] Post Discord Login

### ISSUE THREAD

**Issue by RockyTheProtogen** [Nyabsi/VRCAA#67]
```
java.lang.NullPointerException: Parameter specified as non-null is null: method cc.sovellus.vrcaa.extension.PreferencesExtensionKt.setDiscordToken, parameter it
	at cc.sovellus.vrcaa.extension.PreferencesExtensionKt.setDiscordToken(Unknown Source:7)
	at cc.sovellus.vrcaa.ui.screen.presence.RichPresenceScreenModel$login$1.invokeSuspend(RichPresenceScreenModel.kt:43)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at android.os.Handler.handleCallback(Handler.java:958)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:230)
	at android.os.Looper.loop(Looper.java:319)
	at android.app.ActivityThread.main(ActivityThread.java:8919)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:578)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [CoroutineName(-7zqfwh:cc.sovellus.vrcaa.ui.screen.presence.RichPresenceScreenModel:default), StandaloneCoroutine{Cancelling}@662db78, Dispatchers.Main.immediate]
```

Details
All I did was log in to Discord, no clue why it's doing this.

**Comment by Nyabsi** [comment]
Hello, that feature has been unmaintained and broken for a while, I will be releasing a fix shortly.

**Comment by Nyabsi** [comment]
Should be fixed by https://github.com/Nyabsi/VRCAA/commit/4775271ca337961e4dc09e3224d86ff8e4e63c88 please let me know if this solves it.

**Comment by RockyTheProtogen** [comment]
> Should be fixed by [4775271](https://github.com/Nyabsi/VRCAA/commit/4775271ca337961e4dc09e3224d86ff8e4e63c88) please let me know if this solves it.

This solves the issue entirely, I will close this issue.
